packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -745,6 +746,7 @@ async function handler(
         const users = await prisma.user.findMany({
           where: {
             username: { in: dynamicUserList },
+            organization: userOrgQuery(req.headers.host.replace(/^https?:\/\//, "")),
```

Comment:
```suggestion
            organization: userOrgQuery(req.headers.host),
```

I don't think host has protocol in it. We seem to be using it as is without replace elsewhere as well.